src/bin/status_icon.rs

```diff
@@ -0,0 +1,91 @@
+//! # TreeView Sample
+//!
+//! This sample demonstrates how to create a StatusIcon.
+
+#![allow(deprecated)]
+
+extern crate gdk_pixbuf;
+extern crate gio;
+extern crate glib;
+extern crate gtk;
+
+use gdk_pixbuf::Pixbuf;
+use gio::MemoryInputStream;
+use glib::Bytes;
+use gtk::{
+    ButtonsType, DialogExt, DialogFlags, Menu, MenuExt, MenuExtManual, MenuItem,
+    MenuItemExt, MenuShellExt, MessageDialog, MessageType, StatusIcon, StatusIconExt, WidgetExt,
+};
+
+use std::rc::Rc;
+use std::cell::RefCell;
+
+fn main() {
+    gtk::init().expect("gtk initialization failed");
+
+    let menu = Menu::new();
+    let dialog_item = MenuItem::new_with_label("Show notification");
+    let close_item = MenuItem::new_with_label("Close app");
+
+    menu.append(&dialog_item);
+    menu.append(&close_item);
+
+    let is_shown = Rc::new(RefCell::new(false));
+
+    let c_is_shown = is_shown.clone();
+    dialog_item.connect_activate(move |_| {
+        *c_is_shown.borrow_mut() = false;
+        let parent: Option<&gtk::Window> = None; // stupid hack to avoid rustc error
```

Comment:
Or: `None::<&gtk::Window>`

Reply:
I \*think\* that I had an issue with this syntax. I'll recheck just to be sure.

Reply:
Fewer letters if it works :)

Reply:
We all agree on that haha.